### PR TITLE
[outbound calling] remove 1-n-calling cc

### PR DESCRIPTION
**Pull request by dmceachernmsft**
# What

remove the one-to-n-calling conditional compile comments
# Why

code cleanup after one-to-n was released

https://skype.visualstudio.com/SPOOL/_workitems/edit/3882907
# How Tested

Built locally CI to validate screenshots

### CODE REVIEW

File: packages/calling-stateful-client/src/TypeGuards.ts [hunk 1]
```diff
@@ -44,7 +44,6 @@ export const _isTeamsCallAgent = (callAgent: CallAgentCommon): callAgent is Team
  * @internal
  */
 export const _isTeamsIncomingCall = (call: IncomingCallCommon): boolean => {
-  /* @conditional-compile-remove(one-to-n-calling) */
   return call.kind === 'TeamsIncomingCall';
   return false;
```
Comment (JamesBurnside) [inline packages/calling-stateful-client/src/TypeGuards.ts:48]: remove unreachable code

File: packages/calling-stateful-client/src/TypeGuards.ts [hunk 2]
```diff
@@ -53,7 +52,6 @@ export const _isTeamsIncomingCall = (call: IncomingCallCommon): boolean => {
  * @internal
  */
 export const _isACSIncomingCall = (call: IncomingCallCommon): boolean => {
-  /* @conditional-compile-remove(one-to-n-calling) */
   return call.kind === 'IncomingCall';
   return false;
```
Comment (JamesBurnside) [inline packages/calling-stateful-client/src/TypeGuards.ts:56]: remove unreachable code

File: packages/react-components/src/components/VideoGallery/utils/videoGalleryLayoutUtils.ts [hunk 3]
```diff
@@ -52,13 +52,12 @@ const getOrganizedParticipants = (props: OrganizedParticipantsArgs): OrganizedPa
...
-  /* @conditional-compile-remove(one-to-n-calling) */
   const callingParticipants = remoteParticipants.filter((p) => p.state === ('Connecting' || 'Ringing'));
-  /* @conditional-compile-remove(one-to-n-calling) */
+
   const callingParticipantsSet = new Set(callingParticipants.map((p) => p.userId));
 
   let connectedParticipants = remoteParticipants;
-  /* @conditional-compile-remove(one-to-n-calling) */
+
   connectedParticipants = connectedParticipants.filter((p) => !callingParticipantsSet.has(p.userId));
```
Comment (JamesBurnside) [inline packages/react-components/src/components/VideoGallery/utils/videoGalleryLayoutUtils.ts:61]: combine with line above and make const

File: packages/react-composites/src/composites/CallComposite/components/HoldPane.tsx [hunk 4]
```diff
@@ -106,7 +102,6 @@ export const getReadableTime = (time: number): string => {
 };
 
 const stringsTrampoline = (locale: CompositeLocale): HoldPaneStrings => {
```
Comment (JamesBurnside) [inline packages/react-composites/src/composites/CallComposite/components/HoldPane.tsx:104]: remove trampoline function

File: packages/react-composites/src/composites/CallWithChatComposite/CallWithChatComposite.tsx [hunk 5]
```diff
@@ -736,7 +736,6 @@ export const CallWithChatComposite = (props: CallWithChatCompositeProps): JSX.El
 };
 
 const hasJoinedCallFn = (page: CallCompositePage, callStatus: CallState): boolean => {
-  /* @conditional-compile-remove(one-to-n-calling) */ /* @conditional-compile-remove(one-to-n-calling) */
   return (
```
Comment (JamesBurnside) [inline packages/react-composites/src/composites/CallWithChatComposite/CallWithChatComposite.tsx:739]: remove unreachable code (return after this return)

File: packages/react-composites/tests/browser/call/hermetic/PeoplePane.test.ts [hunk 6]
```diff
@@ -25,7 +25,6 @@ import type { MockCallAdapterState } from '../../../common';
 import { toFlatCommunicationIdentifier } from '@internal/acs-ui-common';
 
 const participantListShownAsFlyout = (): boolean => {
-  /* @conditional-compile-remove(one-to-n-calling) */
   return false;
   return true;
```
Comment (JamesBurnside) [inline packages/react-composites/tests/browser/call/hermetic/PeoplePane.test.ts:29]: remove unreachable code

File: packages/react-composites/src/composites/CallComposite/components/CallArrangement.tsx [hunk 7]
```diff
@@ -628,7 +616,7 @@ export const CallArrangement = (props: CallArrangementProps): JSX.Element => {
                       )}
                   </Stack.Item>
                   {renderGallery && props.onRenderGalleryContent && props.onRenderGalleryContent()}
-                  {true && /* @conditional-compile-remove(one-to-n-calling) */ !isInLocalHold && (
+                  {true && !isInLocalHold && (
```
Comment (JamesBurnside) [inline packages/react-composites/src/composites/CallComposite/components/CallArrangement.tsx:619]: remove `true`

**Review by JamesBurnside — APPROVED** [review]

**Review by mgamis-msft — APPROVED** [review]

**Pull request merged**